Description: fix encoding issues in test_build_html
 test_build_html: open files that are fed to ElementTree parser in "rb"
 mode, fixing encoding issues.
Origin: upstream, https://bitbucket.org/birkenfeld/sphinx/changeset/15c9d212bbf4
Bug: https://bitbucket.org/birkenfeld/sphinx/issue/895

--- a/tests/test_build_html.py
+++ b/tests/test_build_html.py
@@ -328,7 +328,7 @@
     for fname, paths in HTML_XPATH.iteritems():
         parser = NslessParser()
         parser.entity.update(htmlentitydefs.entitydefs)
-        fp = open(os.path.join(app.outdir, fname))
+        fp = open(os.path.join(app.outdir, fname), 'rb')
         try:
             etree = ET.parse(fp, parser)
         finally:
